<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(33, 33, 33)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="transportTableWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>439</y>
      <width>640</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>208</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>60</number>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
     <property name="font">
      <font>
       <family>Noto Serif CJK TC Black</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </row>
    <column>
     <property name="text">
      <string>路線</string>
     </property>
     <property name="font">
      <font>
       <family>Noto Serif CJK TC Black</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </column>
    <column>
     <property name="text">
      <string>車牌號碼</string>
     </property>
     <property name="font">
      <font>
       <family>Noto Serif CJK TC Black</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </column>
    <column>
     <property name="text">
      <string>到站剩餘時間</string>
     </property>
     <property name="font">
      <font>
       <family>Noto Serif CJK TC Black</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>1815</string>
     </property>
     <property name="font">
      <font>
       <family>Noto Serif CJK TC Black</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>進站中</string>
     </property>
     <property name="font">
      <font>
       <family>Noto Serif CJK TC Black</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="datetimeLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif CJK TC Black</family>
      <pointsize>90</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>2021/08/12 23:59:59</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>1</number>
    </property>
   </widget>
   <widget class="QTableWidget" name="toDoTableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>169</y>
      <width>640</width>
      <height>631</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>312</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>60</number>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
     <property name="font">
      <font>
       <family>Noto Serif CJK TC Black</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </row>
    <column>
     <property name="text">
      <string>描述</string>
     </property>
     <property name="font">
      <font>
       <family>Noto Serif CJK TC Black</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </column>
    <column>
     <property name="text">
      <string>到期日</string>
     </property>
     <property name="font">
      <font>
       <family>Noto Serif CJK TC Black</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string> 戴口罩</string>
     </property>
     <property name="font">
      <font>
       <family>Noto Serif CJK TC Black</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>08/31</string>
     </property>
     <property name="font">
      <font>
       <family>Noto Serif CJK TC Black</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="heartbeatTableWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>169</y>
      <width>640</width>
      <height>271</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>312</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>60</number>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
     <property name="font">
      <font>
       <family>Noto Serif CJK TC Black</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </row>
    <column>
     <property name="text">
      <string>名稱</string>
     </property>
     <property name="font">
      <font>
       <family>Noto Serif CJK TC Black</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </column>
    <column>
     <property name="text">
      <string>狀態</string>
     </property>
     <property name="font">
      <font>
       <family>Noto Serif CJK TC Black</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>Raspi-Palor</string>
     </property>
     <property name="font">
      <font>
       <family>Noto Serif CJK TC Black</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>回應正常</string>
     </property>
     <property name="font">
      <font>
       <family>Noto Serif CJK TC Black</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
